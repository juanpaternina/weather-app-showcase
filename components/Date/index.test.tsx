import { render, screen } from '@testing-library/react-native';

const { Date } = require('.');

describe('Date', () => {
  test('It should render a Text component', () => {
    const { toJSON } = render(<Date>Date</Date>);
    expect(toJSON()).toMatchSnapshot();

    const text = screen.getByText('Date');
    expect(text).toBeDefined();
  });
});
