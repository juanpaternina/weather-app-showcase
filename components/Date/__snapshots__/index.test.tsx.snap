// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date It should render a Text component 1`] = `
<Text
  style={
    {
      "alignSelf": "center",
      "backgroundColor": "black",
      "borderBottomLeftRadius": 32,
      "borderBottomRightRadius": 32,
      "borderTopLeftRadius": 32,
      "borderTopRightRadius": 32,
      "color": "white",
      "fontFamily": "medium",
      "fontSize": 16,
      "fontWeight": "normal",
      "minWidth": 200,
      "paddingHorizontal": 14,
      "paddingVertical": 5,
      "textAlign": "center",
    }
  }
>
  Date
</Text>
`;
